swagger.yaml

```diff
@@ -288,6 +288,73 @@ paths:
 
         404: 
           description: The requested transponder is not currently live
+          
+  '/events/{event_slug}/live/leaderboard.html':
```

Comment:
This endpoint supports both a json and rss response. We need to add a response format for each as well as how to request the rss response instead of json.